main.py

```diff
@@ -312,6 +315,10 @@ def search_key():
                         text=basicFont.render('BING!',True,ORANGE,BLACK)
                         pygame.mixer.music.load('./sounds/win.mp3')
                         pygame.mixer.music.play(0)
+                        pygame.time.delay(3200)
+                        pygame.mixer.music.load('./sounds/Sherlock Bbc.mp3')
```

Comment:
We shouldn't use Sherlock theme as our sound, take another sound which won't get us in trouble if we use it :p check out some free sounds to use if there are good ones